src/components/platform/abous-us/about-us.scss

```diff
@@ -0,0 +1,13 @@
+.table1 {
+    width: 20%;
+}
+
+.table1 td, th {
+    border: 1px solid transparent;
```

Comment:
i assume you did transparent here to have a little spacing between the cells. if you want to do that, you could also do "border-spacing: 1px", which would be applied to the table element.